src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java

```diff
@@ -53,6 +53,34 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;EmptyCatchBlock&quot;/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * try {
+ *
+ * }
+ * catch(Exception e) {
+ *
+ * } // violation
+ * </pre>
+ * <p>
+ * Example to show that check allows  empty catch block with any comment inside:
+ * </p>
+ * <pre>
+ * try {
+ *
+ * }
+ * catch(Exception e) { // Comments allowed inside empty catch block
```

Comment:
1. `// Comments allowed inside empty catch block` please make it more look like user code and move comment to next line
2. make it something like `// no handling`
3. extend ok comment with explanation to make it something like `// ok, catch block has comment`